DAO/project/contracts/DAO-contract/tests/utils/setup.rs

```diff
@@ -32,16 +34,41 @@ pub(crate) fn proposal_transaction(asset_id: ContractId) -> Proposal {
 }
 
 pub(crate) async fn setup() -> (ContractId, ContractId, Metadata, Metadata, u64) {
-    let (mut wallets, mut asset_ids) = setup_wallets().await;
+    let mut rng = rand::thread_rng();
+    let num_coins = 1;
+    let coin_amount = 1_000_000;
+
+    let base_asset = AssetConfig {
+        id: BASE_ASSET_ID,
+        num_coins,
+        coin_amount,
+    };
+    let mut gov_token_id = AssetId::zeroed();
+    gov_token_id.try_fill(&mut rng).unwrap();
```

Comment:
The randomness can be replaced by creating the `AssetId` using the `::from(array)` syntax

Reply:
I don't mind making this change, and understand that generally it's not desirable to introduce entropy to the test suite but in this case I'm unsure why the entropy is an issue when creating wallet addresses; the [rust SDK suggests this in the docs](https://rust.fuel.network/v0.44.0/wallets/test-wallets.html#setting-up-a-test-wallet-with-multiple-custom-assets), and the [rust SDK's functions](https://docs.rs/fuels-test-helpers/0.44.0/src/fuels_test_helpers/lib.rs.html#50-55) also use `rand`.

Reply:
It's unnecessary. Adding entropy when it's not needed is akin to adding complexity when it's not needed. Things may go wrong and it may be difficult to debug. If you can make it static then it should be static.